examples/nestjs/src/BUILD.bazel

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+
+ts_library(
+    name = "app",
+    srcs = glob(["*.ts"]),
+    deps = [
+        "@npm//@nestjs/common",
+        "@npm//@nestjs/core",
+        "@npm//@types/node",
+        "@npm//tslib",
+    ],
+)
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
+
+# bazel run //src:server -- --port=4000
+nodejs_binary(
+    name = "server",
+    data = [
+        ":app",
+        "@npm//@nestjs/common",
+        "@npm//@nestjs/core",
+        "@npm//@nestjs/platform-express",
+        "@npm//minimist",
+    ],
+    entry_point = ":main.ts",
+)
+
+load("@io_bazel_rules_docker//nodejs:image.bzl", "nodejs_image")
+
+# bazel build --platforms=@build_bazel_rules_nodejs//toolchains/node:linux_amd64 //src:docker
+nodejs_image(
+    name = "docker",
+    data = [
+        ":app",
+    ],
+    entry_point = ":main.ts",
+    node_modules = "@npm//:node_modules",
```

Comment:
To be honest we should update rules_docker to not require this anymore so we just use fine-grained deps. Otherwise the image can really blow up in size. I will get a PR ready shortly for that. My workaround (and also from other users I have read) is to point this to an empty filegroup and just use fine-grained deps.

Reply:
😎

Reply:
See: https://github.com/bazelbuild/rules_docker/pull/1000